This modification has been introduced in revision 3616 but crashes LBA 2,
 which uses a CHDIR C:.

Signed-off-by: Laurent Merckx <laurent-merckx@skynet.be>

--- dosbox/src/dos/dos_files.cpp.o	2016-07-06 14:21:34.929300170 +0200
+++ dosbox.n/src/dos/dos_files.cpp	2016-07-06 14:21:46.009300114 +0200
@@ -206,7 +206,7 @@ bool DOS_GetCurrentDir(Bit8u drive,char
 bool DOS_ChangeDir(char const * const dir) {
 	Bit8u drive;char fulldir[DOS_PATHLENGTH];
 	const char * testdir=dir;
-	if (strlen(testdir) && testdir[1]==':') testdir+=2;
+	/*if (strlen(testdir) && testdir[1]==':') testdir+=2;*/
 	size_t len=strlen(testdir);
 	if (!len) {
 		DOS_SetError(DOSERR_PATH_NOT_FOUND);
